images/devtools-kubernetes-v1beta1/context/magefiles/template.go

```diff
@@ -0,0 +1,99 @@
+//go:build mage
+// +build mage
+
+package main
+
+import (
+	"fmt"
+	"github.com/magefile/mage/sh" // sh contains helpful utility functions, like RunV
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+func renderTemplate(app ArgoCDApp) (string, error) {
+	if app.Spec.Source.Helm.ReleaseName != "" {
+		return renderHelm(app.Spec.Source)
+	} else if _, err := os.Stat(app.Spec.Source.Path + "/kustomize.yaml"); err == nil {
```

Comment:
Should be `kustomization.yaml`

```suggestion
	} else if _, err := os.Stat(app.Spec.Source.Path + "/kustomization.yaml"); err == nil {
```

This silently failed for me, and then ran validation against unbuilt templates which broke in confusing ways.